src/view/components/formElementContainer.styl

```diff
@@ -0,0 +1,7 @@
+.FormElementContainer > * {
+  margin-top: var(--spectrum-global-dimension-size-100);
+}
+
+.FormElementContainer > :first-child {
+  margin-top: 0;
+}
```

Comment:
I don't know why I didn't catch this when I reviewed the PR, but the proper way to do this would be to use the `gap` prop on the `Flex` component with a value of `size-100` (which in turn sets the `gap` CSS attribute). Feel free to change it right on the react-spectrum branch or I'll change it when I merge this stuff into my working changes.